models/cart-discount/src/cart-discount-draft/transformers.ts

```diff
@@ -0,0 +1,27 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TCartDiscountDraft, TCartDiscountDraftGraphql } from '../types';
+
+const transformers = {
+  default: Transformer<TCartDiscountDraft, TCartDiscountDraft>('default', {
+    buildFields: ['value', 'name', 'description'],
+  }),
+  rest: Transformer<TCartDiscountDraft, TCartDiscountDraft>('rest', {
+    buildFields: ['value', 'name', 'description'],
+  }),
+  graphql: Transformer<TCartDiscountDraft, TCartDiscountDraftGraphql>(
+    'graphql',
+    {
+      buildFields: ['value', 'name', 'description'],
+      addFields: ({ fields }) => ({
+        nameAllLocales: LocalizedString.toLocalizedField(fields.name),
+        descriptionAllLocales: LocalizedString.toLocalizedField(
+          fields.description
+        ),
```

Comment:
I don't see these fields in the GraphQL docs
![image](https://user-images.githubusercontent.com/8136478/217640082-e3034012-5fbd-4d33-843b-3d9b50f69484.png)

Reply:
Resolved in fa1ff8648a9f7b779e0e292bdd12fe0245538fb1